star-rx72n-firmware/lib/rx_usb_comm/src/rx_usb_comm.c

```
@@ -754,6 +754,147 @@ rx_usb_comm_send_nack(rx_usb_comm_handle_t* handle, const uint16_t sequence, con
+/**
+ * @brief Handle control frames (PING, RESET) internally
+ *
+ * PING: Auto-sends PONG echoing the payload, invokes on_ping_cb.
+ * RESET: Resets sequence counters, auto-sends RESET_ACK, invokes on_reset_cb.
+ * All other frame types pass through to the caller.
+ */
+static usb_dispatch_result_t internal_dispatch_control_usb(rx_usb_comm_handle_t* handle,
+                                                            const rx_frame_t*     frame,
+                                                            rx_err_t*             err)
+{
+  const rx_frame_type_t type = (rx_frame_type_t)frame->header.type;
```

Clang (14.0.6)

[warning] variable 'type' is not initialized

(cppcoreguidelines-init-variables)